src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -5,8 +5,10 @@ CHANGELOG
 -----
 
  * added `AppVariable::getToken()`
- * Deprecated the possibility to inject the Form Twig Renderer into the form
-   extension. Inject it on TwigRendererEngine instead.
+ * [BC BREAK] Deprecated the possibility to inject the Form `TwigRenderer` into the form extension. 
```

Comment:
I would also add the code example to the changelog. In fact, you will probably only need the example if you are not using the full-stack framework. But then, you will also not have the upgrade files, but read the changelog instead.